pom.xml

```diff
@@ -126,6 +128,11 @@
 			</dependency>
 			<dependency>
 				<groupId>org.springframework.cloud</groupId>
+				<artifactId>spring-cloud-common-security-config-starter-web</artifactId>
```

Comment:
I was a bit too quick to merge that rename pr in spring-cloud-common-security-config as now that I look at this name, it's wrong. If using convention then it should be `spring-cloud-starter-common-security-config-web` or any other shorter combination which makes sense like `spring-cloud-starter-common-security`. It just needs to start with `spring-cloud-starter-`.

Reply:
@jvalkeal we can have that name change if that's the convention. The name `spring-cloud-starter-common-security` is generic and that's the reason, we added `common-security-config-web` to work on commong web security configuration. Are you ok with the name `spring-cloud-starter-common-security-config-web` (apart from it being long  ;)) ?